gratipay/models/participant/email.py

```diff
@@ -177,16 +178,20 @@ def get_email_verification_nonce(self, c, email):
 
     def start_package_claims(self, c, nonce, *packages):
         """Takes a cursor, nonce and list of packages, inserts into ``claims``
-        and returns ``None``.
+        and returns ``None`` (or raise :py:exc:`NoPackages`).
         """
         if not packages:
-            return
-        VALUES, values = [], []
+            raise NoPackages()
+
+        # We want to make a single db call to insert all claims, so we need to
+        # do a little SQL construction. Do it in such a way that we still avoid
+        # Python string interpolation (~= SQLi vector).
+
+        extra_sql, values = '', []
         for p in packages:
-            VALUES.append('(%s, %s)')
+            extra_sql += ' (%s, %s)'
```

Comment:
Looks like maybe this isn't going to be delimited by commas anymore?

" (%s, %s) (%s, %s) (%s, %s)"

Maybe that's OK for Postgres though?